### PR TITLE
Changed Hardcoded text for Magento Commerce Cloud to Variable

### PR DESCRIPTION
**Pull request by BarnyShergold**
## Purpose of this pull request

This pull request (PR) replaced all hardcoded instances of the text "Magento Commerce Cloud" with the system variable {{ site.data.var.ece }} in files where appropriate. This makes for consistency of naming and also eliminates spelling mistakes (one page had Commerce spelt incorrectly 

## Affected DevDocs pages
https://devdocs.magento.com/cloud/reference/ece-tools-error-reference.html
https://devdocs.magento.com/cloud/integrations/gitlab-integration.md
https://devdocs.magento.com/cloud/project/services-mysql.md
https://devdocs.magento.com/cloud/project/user-admin.md
https://devdocs.magento.com/cloud/release-notes/ece-release-notes.md
https://devdocs.magento.com/cloud/requirements/cloud-requirements.md
https://devdocs.magento.com/community/resources/best-practices.md
https://devdocs.magento.com/guides/v2.3/comp-mgr/patching/mqp.md
https://devdocs.magento.com/guides/v2.3/config-guide/prod/config-reference-var-name.md
https://devdocs.magento.com/guides/v2.3/release-notes/release-notes-2-3-5-commerce.md
https://devdocs.magento.com/guides/v2.3/release-notes/release-notes-2-3-5-open-source.md
https://devdocs.magento.com/guides/v2.4/release-notes/commerce-2-4-1.md
https://devdocs.magento.com/guides/v2.4/release-notes/release-notes-2-4-0-commerce.md
https://devdocs.magento.com/guides/v2.4/release-notes/release-notes-2-4-0-open-source.md

### CODE REVIEW

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
running tests

File: src/guides/v2.3/config-guide/prod/config-reference-var-name.md [hunk 1]
```diff
@@ -242,7 +242,7 @@ content='
 
 *  To use values you set in the `$_ENV` array, you must set `variables_order = "EGPCS"` in your `php.ini` file. For details, see [PHP documentation](http://us.php.net/manual/en/ini.core.php#ini.variables-order).
 
-*  For Magento Commerce Cloud, if you are attempting to override Magento configuration settings using the [Project Web Interface](https://devdocs.magento.com/cloud/project/project-webint-basic.html#project-conf-env-var), you must prepend the variable name with `env:`. For example:
+*  For {{ site.data.var.ece }}, if you are attempting to override Magento configuration settings using the [Project Web Interface](https://devdocs.magento.com/cloud/project/project-webint-basic.html#project-conf-env-var), you must prepend the variable name with `env:`. For example:
```
Comment (dshevtsov) [inline src/guides/v2.3/config-guide/prod/config-reference-var-name.md]: Revert this change. Liquid variables are not allowed within Liquid tags. This one is within  `{% include note.html ... %}`.

**Review by dshevtsov — CHANGES_REQUESTED** [review]

**Review by dshevtsov — APPROVED** [review]

**Comment by dobooth** [comment]
running tests
